core/src/it/scala/org/ensime/core/JavaCompilerSpec.scala

```diff
@@ -113,7 +113,7 @@ class JavaCompilerSpec extends EnsimeSpec
           case "3" =>
             info.name shouldBe "java.io.PrintStream.println(java.lang.Object)"
             info.localName shouldBe "println"
-            info.`type`.name shouldBe "(java.lang.Object)void"
+            info.`type`.name shouldBe "public void println(Object arg0)"
```

Comment:
this is more like the symbol at point rather than the type at point. The type wouldn't have the `println` name in it.

Seems we need a symbol-at-point and a type-at-point distinction in java.